### ISSUE TITLE
Correct input handling for aeif_cond_alpha_multisynapse

### ISSUE THREAD

**Issue by heplesser** [nest/nest-simulator#509]
Input handling for `aeif_cond_alpha_multisynapse` needs to be modified according to the decision made in the [NEST Developer VC 19 Sep 2016](https://github.com/nest/nest-simulator/wiki/2016-09-19-Open-NEST-Developer-Video-Conference):
- Replace `E_ex`, `E_in` with vector of reversal potentials the same size as `taus_syn` (one per synapse)
- Replace  `I_syn_ex`, `I_syn_in` recordables with one recordable `I_syn` representing the total input current
- Update class and `handle(SpikeEvent&)` to a single input buffer per rport, see [`aeif_cond_beta_multisynapse`](https://github.com/nest/nest-simulator/blob/446d340da0830edcc78ac85ae0a80e58695b3e37/models/aeif_cond_beta_multisynapse.cpp#L607) for an example; consider adding a check for non-negative weight.
- Update `*_dynamics` function to handle input current correctly, along the lines of

``` C++
  double I_syn = 0.0;
  for ( size_t i = 0; i < node.P_.num_of_receptors_ ; ++i )
  {
      I_syn += y[ S::G + i ] * ( node.P_.E_rev[i] - V );
  }
```
- Move `*_dynamics` function to cpp-file.
- Provide examples (python based).

See also #510, #511.

**Comment by heplesser** [comment]
This is now implemented in PR #557.

**Comment by heplesser** [comment]
Fixed by #557.
